docs/packages/universal-components/contributing.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Contributing guidelines - universal-components
+sidebar_label: Contributing
+---
+
+_Note: These guidelines assume you have an existing React Native setup on your machine and that your simulator/emulator is properly functionning. If not, have a look at [Getting Started • React Native](https://facebook.github.io/react-native/docs/getting-started.html), and go to "Building Projects with Native Code"._
+
+## Storybook
+
+To run the storybook and start developing, clone the repository
+
+```bash
+git clone https://github.com/kiwicom/margarita.git
+```
+
+and enter the directory to install the node modules:
+
+```bash
+cd margarita
+yarn
+```
+
+### Desktop
+
+If you want to see the web version, run
+
+```bash
+yarn storybook:web:universal-components
+```
+
+and you should go to <http://localhost:9009> to see
+
+![](./docs/assets/storybook-web.png)
```

Comment:
This asset seems to be missing. Same for `storybook-native.png` on row 53.

Reply:
Good shout! Thanks!